src/pytest_inmanta_lsm/remote_orchestrator.py

```diff
@@ -44,7 +44,10 @@ def __init__(
         project: Project,
         settings: Dict[str, Union[bool, str, int]],
         noclean: bool,
+        token: Optional[str],
+        ca_cert: Optional[str],
         ssh_port: str = "22",
+        ssl: bool = False,
```

Comment:
This is a breaking change on the API of the `RemoteOrchestrator` class. I am not sure whether we use this object in other places than the `pytest-inmanta-lsm` library. It's probably best to play it safe and to make sure that this change is backwards compatible.
```suggestion
        ssh_port: str = "22",
        token: Optional[str] = None,
        ca_cert: Optional[str] = None,
        ssl: bool = False,
```